src/vdaf/poplar1.rs

```diff
@@ -28,22 +28,29 @@ const DST_CORR_INNER: u16 = 2;
 const DST_CORR_LEAF: u16 = 3;
 const DST_VERIFY_RANDOMNESS: u16 = 4;
 
-/// Poplar1 with [`PrgAes128`].
-pub type Poplar1Aes128 = Poplar1<PrgAes128, 16>;
-
-impl Poplar1Aes128 {
-    /// Create an instance of [`Poplar1Aes128`]. The caller provides the bit length of each
+impl<P, const L: usize> Poplar1<P, L> {
+    /// Create an instance of [`Poplar1`]. The caller provides the bit length of each
     /// measurement (`BITS` as defined in the [[draft-irtf-cfrg-vdaf-03]]).
     ///
     /// [draft-irtf-cfrg-vdaf-03]: https://datatracker.ietf.org/doc/draft-irtf-cfrg-vdaf/03/
-    pub fn new_aes128(bits: usize) -> Self {
+    pub fn new(bits: usize) -> Self {
         Self {
             bits,
             phantom: PhantomData,
         }
     }
 }
 
+impl Poplar1<PrgSha3, 16> {
+    /// Create an instance of [`Poplar1`] using [`PrgSha3`]. The caller provides the bit length of
+    /// each measurement (`BITS` as defined in the [[draft-irtf-cfrg-vdaf-03]]).
+    ///
+    /// [draft-irtf-cfrg-vdaf-03]: https://datatracker.ietf.org/doc/draft-irtf-cfrg-vdaf/03/
+    pub fn new_sha3(bits: usize) -> Self {
```

Comment:
This is the most awkward naming choice I had to make -- I wanted to have one generic constructor, for BYO-PRG consumers, and one non-generic constructor, for general convenience, but they can't both be `new()`. Hopefully the parallelism break isn't too bad.

Reply:
This is a little awkward, but not too bad IMO.